### PR TITLE
added section for Onboarding (now called "Helping Users getting Started")

### PR DESCRIPTION
**Pull request by pbrunschwig**

### CODE REVIEW

File: doc/level1.rst [hunk 1]
```diff
@@ -878,12 +878,43 @@ Level 1 MUAs MUST allow the user to disable Autocrypt completely for
...
 Autocrypt disabled by default.
 
-Onboarding
-++++++++++
 
-.. todo::
+Helping Users getting Started
++++++++++++++++++++++++++++++
+
+The following paragraph provides recommendations for MUA
```
Comment (dkg) [inline doc/level1.rst]: paragraph?

File: doc/level1.rst [hunk 2]
```diff
@@ -878,12 +878,43 @@ Level 1 MUAs MUST allow the user to disable Autocrypt completely for
...
+
+2. If a sent message with an Autocrypt header was found:
+
+   Provide guidance for creating an Autocrypt Setup Message
+   on the MUA that created the message.
+
+3. If there is evidence of actively used OpenPGP software
+   (for example if a secret key is available, some
+   specific software is installed, etc.) or if encrypted
+   mails are found:
```
Comment (dkg) [inline doc/level1.rst]: Is point 3 here intended to mean "non-autocrypt" only?  or would encrypted messages from autocrypt software also trigger this point?

File: doc/level1.rst [hunk 3]
```diff
@@ -878,12 +878,43 @@ Level 1 MUAs MUST allow the user to disable Autocrypt completely for
...
+
+The MUA SHOULD scan the mailbox for sent mails (wherever
+they might be) that show evidence of OpenPGP/Autocrypt
+usage.
+
+
+1. If an Autocrypt Setup Message was found:
+
+   Start a setup process suggesting the user to import the
+   settings.
```
Comment (dkg) [inline doc/level1.rst]: s/settings/setup message/, right?

File: doc/level1.rst [hunk 4]
```diff
@@ -878,12 +878,43 @@ Level 1 MUAs MUST allow the user to disable Autocrypt completely for
...
+
+The following paragraph provides recommendations for MUA
+implementations to help users start Autocrypt after an
+account was set up.
+
+The MUA SHOULD scan the mailbox for sent mails (wherever
+they might be) that show evidence of OpenPGP/Autocrypt
+usage.
+
+
```
Comment (dkg) [inline doc/level1.rst]: Is the series below a "first-match-wins" situation?  or should the MUA try them all in parallel?

If the search takes a long time, do we need to exhaust the search first, and then decide what to do based on what we find, or should we take these actions as we find stuff?

File: doc/level1.rst (hunk 2 above)
Comment (pbrunschwig) [reply doc/level1.rst]: The numbered list is intended as sequence to follow for all found message. You need to keep a state for it.

**Pull request merged**